Userland/Libraries/LibWeb/Streams/ReadableStream.cpp

```diff
@@ -14,9 +14,9 @@ namespace Web::Streams {
 // https://streams.spec.whatwg.org/#rs-constructor
 WebIDL::ExceptionOr<JS::NonnullGCPtr<ReadableStream>> ReadableStream::construct_impl(JS::Realm& realm)
 {
-    auto* readable_stream = realm.heap().allocate<ReadableStream>(realm, realm);
+    auto readable_stream = realm.heap().allocate<ReadableStream>(realm, realm);
 
-    return JS::NonnullGCPtr { *readable_stream };
+    return readable_stream;
```

Comment:
Seems like this should just be a single line now, but not a big deal

Reply:
done